src/components/Filter/utils.ts

```diff
@@ -70,8 +75,18 @@ export const getHoverBorder = ({
     return `transparent`;
   }
   if (styleType === 'outlined' || hasSelectedValue) {
-    return `${stateBackgroundColor(theme, 'hover', calculatedColor, true)}`;
+    return `${getHoverBackgroundColor(theme, calculatedColor)}`;
   }
 
   return 'transparent';
 };
+
+export const getHoverBackgroundColor = (
```

Comment:
consider use this function `stateBackgroundColor` as the logic is already there

Reply:
thank you! 
`stateBackgroundColor` was initially used here but the hover background color was not the one we wanted, as inside `stateBackgroundColor` the background color for hover shade based on the primary or secondary color but here we need lighten or darken of the default color (that is primary color with shade 100)

Maybe I miss something tho, so if `stateBackgroundColor` can be used here, could you explain how? thanks again

Reply:
@panvourtsis I have created an issue [here](https://github.com/Orfium/orfium-ictinus/issues/349), because i need this PR merged for the current sprint. Let' s discuss the improvement there.

Thank you!